styles/blocks/logo.scss

```diff
@@ -0,0 +1,12 @@
+@use '../utils/mixins' as *;
+
+.logo {
+
+  &__image {
+    height: 16px;
+
+    @include for-mobile-big() {
+      height: 24px;
+    }
+  }
+}
\ No newline at end of file
```

Comment:
It is recommended to always have a newline at the end of a file. This is a traditional UNIX standard that ensures easy concatenation of files and better compatibility across different editors and tools.